packages/shared/src/components/modals/streaks/StreakRecoverModal.tsx

```diff
@@ -0,0 +1,207 @@
+import React, { ReactElement, useId } from 'react';
+import { ModalSize } from '../common/types';
+import { ModalBody } from '../common/ModalBody';
+import { Modal, ModalProps } from '../common/Modal';
+import { LoggedUser } from '../../../lib/user';
+import {
+  Typography,
+  TypographyTag,
+  TypographyType,
+} from '../../typography/Typography';
+import {
+  Button,
+  ButtonProps,
+  ButtonSize,
+  ButtonVariant,
+} from '../../buttons/Button';
+import { anchorDefaultRel } from '../../../lib/strings';
+import { reputation as reputationHref } from '../../../lib/constants';
+import { ReputationIcon } from '../../icons';
+import { IconSize } from '../../Icon';
+import {
+  useStreakRecover,
+  UseStreakRecoverReturn,
+} from '../../../hooks/streaks/useStreakRecover';
+import { Checkbox } from '../../fields/Checkbox';
+import { ModalClose } from '../common/ModalClose';
+import { cloudinary } from '../../../lib/image';
+
+export interface StreakRecoverModalProps
+  extends Pick<ModalProps, 'isOpen' | 'onAfterClose'> {
+  onRequestClose: () => void;
+  user: LoggedUser;
+}
+
+const StreakRecoverCover = () => (
+  <div className="overflow-hidden tablet:-mx-4" role="presentation" aria-hidden>
+    <img
+      alt="Broken reading streak"
+      className="h-auto w-full object-contain"
+      height="131"
```

Comment:
We don't have a design for mobile, but added width/height to prevent layout shift once the image is loading (it is lazy for now for performance). Should i remove them?

Reply:
It just sounded like probably more prone to issues down the line then having it dynamic.
(Which we do now everywhere else from what I can see)

CC: @nensidosari Not really having a strong opinion, just want to avoid accidentally missing some view)

Reply:
True, we never set them usually. I would suggest to remove for now, if we see layout shift is becoming an issue in this case we can consider it